crates/polars-plan/src/dsl/function_expr/range/int_range.rs

```diff
@@ -26,78 +28,6 @@ pub(super) fn int_range(s: &[Series], step: i64) -> PolarsResult<Series> {
     }
 }
 
-pub(super) fn int_ranges(s: &[Series], step: i64) -> PolarsResult<Series> {
-    let start = &s[0].rechunk();
-    let end = &s[1].rechunk();
-
-    let output_name = "int_range";
-
-    let mut start = start.cast(&DataType::Int64)?;
-    let mut end = end.cast(&DataType::Int64)?;
-
-    if start.len() != end.len() {
-        if start.len() == 1 {
-            start = start.new_from_index(0, end.len())
-        } else if end.len() == 1 {
-            end = end.new_from_index(0, start.len())
-        } else {
-            polars_bail!(
-                ComputeError:
-                "lengths of `start`: {} and `end`: {} arguments `\
-                cannot be matched in the `int_ranges` expression",
-                start.len(), end.len()
-            );
-        }
-    }
-
-    let start = start.i64()?;
-    let end = end.i64()?;
-
-    let start = start.downcast_iter().next().unwrap();
-    let end = end.downcast_iter().next().unwrap();
-
-    // First do a pass to determine the required value capacity.
```

Comment:
I removed this pass and set a default capacity of 5 times the length of the column, similar to the other `ranges` functions. If this capacity calculation must be preserved, I have to make some adjustments as it relied on fully materializing start/end.